src/ModularPipelines/Helpers/ProgressPrinter.cs

```diff
@@ -173,7 +173,7 @@ private static string GetModuleExtraInformation(ModuleBase module)
                 var ticksPerSecond = 100 / totalEstimatedSeconds;
```

Comment:
Could you also turn this into 100.0? The fact it's an int is probably causing rounding issues which might be the reason for this.

Reply:
done!